docs/reference/system/utilities.md

```diff
@@ -8,110 +8,176 @@ pageClass: page-reference
 
 > Utilities are the various helper endpoints located within the API.
 
----
-
 ## Generate a Hash
 
 Generate a hash for a given string.
 
-### Request Body
-
-`string` **Required**\
-String to hash.
-
-### Returns
-
-Hashed string.
-
-### REST API
+### Request
 
-```
-POST /utils/hash/generate
-```
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
 
-##### Example
+`POST /utils/hash/generate`
 
 ```json
-// POST /utils/hash/generate
 
 {
-	"string": "Hello World!"
+	"string": "hash"
 }
 ```
 
-### GraphQL
+</template>
+<template #graphql>
 
-```
-POST /graphql/system
-```
+`POST /graphql/system`
 
 ```graphql
 type Mutation {
 	utils_hash_generate(string: String!): String
 }
 ```
 
-##### Example
+</template>
+</SnippetToggler>
+
+#### Request Body
+
+`string` **Required**\
+String to hash.
+
+### Response
+
+Hashed string.
+
+### Example
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
+
+`POST /utils/hash/generate`
+
+```json
+{
+	"string": "Hello World!"
+}
+```
+
+</template>
+<template #graphql>
 
 ```graphql
 mutation {
 	utils_hash_generate(string: "Hello World!")
 }
 ```
 
----
+</template>
+</SnippetToggler>
 
 ## Verify a Hash
 
 Verify a string with a hash.
 
-### Request Body
+### Request
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
+
+`POST /utils/hash/verify`
+
+```json
+{
+	"string": "test_string",
+	"hash": "hash"
+}
+```
+
+</template>
+<template #graphql>
+
+`POST /graphql/system`
+
+```graphql
+type Mutation {
+	utils_hash_verify(hash: String!, string: String!): Boolean
+}
+```
+
+</template>
+</SnippetToggler>
+
+#### Request Body
 
 `string` **Required**\
 Source string.
 
 `hash` **Required**\
 Hash you want to verify against.
 
-### Returns
+### Response
 
 Boolean.
 
-### REST API
+### Example
 
-```
-POST /utils/hash/verify
-```
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
 
-##### Example
+`POST /utils/hash/verify`
 
 ```json
-// POST /utils/hash/verify
-
 {
 	"string": "Hello World!",
 	"hash": "$arg...fEfM"
 }
 ```
 
-### GraphQL
+</template>
+<template #graphql>
 
-```
-POST /graphql/system
-```
+`POST /graphql/system`
 
 ```graphql
 type Mutation {
-	utils_hash_verify(hash: String!, string: String!): Boolean
+	utils_hash_verify(hash: "$arg...fEfM", string: "Hello World!")
```

Comment:
This seems to be wrong. If the idea is to provide an example, it should be `mutation { ... }` instead.